micro-deps/build.gradle

```diff
@@ -54,14 +51,8 @@ project(':micro-deps:micro-deps-spring-test-config') {
         compile 'org.objenesis:objenesis:2.1'
         compile 'org.aspectj:aspectjweaver'
 
-        //Workaround for a bug in Idea which mixes classpath when running tests - it prevents clashing Spock versions
-        if (System.properties.'idea.active') {
-            compile "org.spockframework:spock-core"
-            compile "org.spockframework:spock-spring"
-        } else {
-            compile "org.spockframework:spock-core:$spock07Version"
-            compile "org.spockframework:spock-spring:$spock07Version"
-        }
+        compile "org.spockframework:spock-core"
+        compile "org.spockframework:spock-spring"
```

Comment:
It is worth to remember to mention about that in release notes. It impacts external developers.